### DISCUSSION TITLE
Is it considered "safe" to make changes to the default `users` collection?

### DISCUSSION THREAD

**Discussion by ericcheatham** [pocketbase/pocketbase#3397] [Q&A] [answered]
I am currently working on a project that is applying some migrations to the default `users` collection. Specifically we want to split `name` into `first_name` and `last_name`. Is it considered "safe" to make changes to this collection?

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, you can remove/change any of the default fields.

**Reply by ericcheatham** [reply]
Awesome. Thank you for the super quick response.
